static/js/panels.js

```diff
@@ -66,6 +66,14 @@ exports.check_profile_incomplete = function () {
     }
 };
 
+function handle_keyup(e) {
+    e.stopPropagation();
+    if (e.keyCode === 13) {
```

Comment:
I'd prefer `e.key === "Enter"` to make it obvious that this is about the enter key.